content/modules/file-input/versions/1.3.0.um

```diff
@@ -0,0 +1,63 @@
+@version 1.3.0
+  @description: A component that extends the native functionality of @code(html)[<input type="file">] to add additional functionality and a hexagon theme
+
+  @examples
+    @example
+      @html
+        <div id="example"></div>
+
+      @js
+        new hx.FileInput('#example')
+
+  @api File Input
+    @added: New File Input component and fluid api function
+      @issue 25
+    @prototype hx.FileInput
+      @description: A component that extends the native functionality of @code(html)[<input type="file">] to add additional functionality and a hexagon theme
+      @constructor
+        @description: Creates a file input using a selector and some options.
```

Comment:
inconsistencies 'file input' vs 'File Input' vs 'FileInput' - maybe stick to FileInput? What do you think?